homeassistant/components/xbox_live/sensor.py

```diff
@@ -10,6 +11,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+SCAN_INTERVAL = timedelta(seconds=60)
```

Comment:
Is the free tier 60 requests per hour per api key or per xbox user profile?

We possibly create many entities per api key that all make requests.

Reply:
🤷‍♂ I just added it based on feedback in multiple places that setting scan_interval to 60 seconds resolved the problem. Hence me opening a PR to change the default (instead of accepting a documentation PR that adds a note).

So what do you suggest? Setting it to 120? 180? 240?

Reply:
If we want it bullet proof we need to change to custom scheduling of updates. The interval should be set according to the number of xuid in the config.

Reply:
Sure, even better, their API sends all kinds of information in their headers (like the number of requests left and stuff), which is even better. However, I don't own an XBox, I don't have XBox Live, and do not plan on code owning this integration.

It is merely a small fix/improvement to what is currently there. Feel free to merge, close or suggest another interval.

Reply:
The documentation is 98% clear that it's per-API, not per-xuid, but I've emailed support to verify.